test/kino/js/live_test.exs

```diff
@@ -25,6 +25,13 @@ defmodule Kino.JS.LiveTest do
       assert count == 1
     end
 
+    test "handle_call/3 :noreply" do
+      kino = LiveCounter.new(0)
+      LiveCounter.bump(kino, 1)
+      count = LiveCounter.read_after(kino, 200)
```

Comment:
This would add 0.2s to the test suit, but any timeout will test the same path :)

```suggestion
      count = LiveCounter.read_after(kino, 0)
```